src/DropDownIcon.jsx

```diff
@@ -106,7 +106,7 @@ DropDownIcon.propTypes = {
   onChange: PropTypes.func,
   menuItems: PropTypes.array.isRequired,
   closeOnMenuItemClick: PropTypes.bool,
-  pushToLeft: PropTypes.bool
+  rePositionToBottomRight: PropTypes.bool
```

Comment:
We already have an anchorOrigin, which means, where to put the popover in relation to the anchor. Why would we need a reposition?